submarine-server/server-rpc/src/main/java/org/apache/submarine/server/rpc/SubmarineRpcServer.java

```diff
@@ -225,7 +224,7 @@ private ApplicationIdProto checkFeature(ParametersHolderProto request) {
 
     protected ApplicationId run(ClientContext clientContext, Parameter parameter)
         throws IOException, YarnException, SubmarineException {
-      // TODO replaced with JobManager
+      // TODO(who) replaced with JobManager
```

Comment:
What do you mean by adding who? Is this generated by code formatter automatically?

Reply:
beacuse todo must specify who to complete.
https://github.com/apache/submarine/blob/master/dev-support/maven-config/checkstyle.xml#L42